### PR TITLE
Add example for deployment command for deploy only js and only css with parallel processing and different deployment strategy.

### PR DESCRIPTION
**Pull request by ravi-chandra3197**
## Purpose of this pull request

This pull request (PR) ...

Add example for deployment command for deploy only js and only css with parallel processing and different deployment strategy.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.html

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md [hunk 1]
```diff
@@ -308,6 +308,16 @@ Successful: 1993 files; errors: 0
 ---
 ```
 
+The following command deploy only js with 4 jobs, deployment strategy is standard
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md:311]:
```suggestion
The following command deploys only JS with 4 jobs, deployment strategy is standard
```

File: src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md [hunk 2]
```diff
@@ -308,6 +308,16 @@ Successful: 1993 files; errors: 0
...
+```bash
+bin/magento setup:static-content:deploy -s standard --no-misc --no-html --no-fonts --no-images --no-less --no-css -j 4
+```
+
+The following command deploy only css and less with 3 jobs, deployment strategy is quick
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md:316]:
```suggestion
The following command deploys only CSS and LESS with 3 jobs, deployment strategy is quick
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few grammatical corrections

File: src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md (hunk 1 above)
Comment (ravi-chandra3197) [reply src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md]: Hello @BarnyShergold 
Thank you for the suggestion, I have update PR, can you review it?

File: src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md (hunk 2 above)
Comment (ravi-chandra3197) [reply src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md]: Hello @BarnyShergold 
Thank you for the suggestion, I have updated PR, can you review it?

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**